README.md

```diff
@@ -157,6 +157,106 @@ This is useful for long examples.
 
 [^back to top](#recog-a-recognition-framework)
 
+### Examples
```

Comment:
Examples are covered earlier in the fingerprints section. Perhaps a heading that indicates this is a section exploring working with matching? Maybe "Matching", "Testing matches"?